external-docs/docs/guides/command-line/inspect-command.md

```diff
@@ -1,3 +1,939 @@
-# Inspect Command
+# inspect command
+The inspect command is used to inspect the metadata of an existing Qube.
 
-todo: Complete me!
\ No newline at end of file
+**Syntax**  
+`csvcubed inspect [OPTIONS] TIDY_CSV_PATH`
+
+**Arguments**
+
+| Argument      | Description                                                                                        |
+| ------------- | -------------------------------------------------------------------------------------------------- |
+| TIDY_CSV_PATH | The file path to the Qube data file, formatted as [tidy data](../glossary/index.md#tidy-data) csv. |
+
+**Options**
+
+| Option      | Description                                                                                                      |
+| ----------- | ---------------------------------------------------------------------------------------------------------------- |
+| --help / -h | Show the command help text.                                                                                      |
+| --log-level | Set the desired logging level to one of 'crit', 'err', 'warn', 'info' and 'debug'.  <br/> The default is 'warn'. |
+
+**Log output**  
+The logs are written to both the console and a log file. The verbosity of the logging is controlled by the `--log-level` option. 
+
+The log messages can be used to help identify and resolve issues that are experienced when inspecting a Qube.
+
+The log path depends on the operating system, and the following are typical paths:
+
+> Below paths may also be influenced by your operating system configuration.
+
+| Operating System | Typical Log Path                                                     |
+| ---------------- | -------------------------------------------------------------------- |
+| **Windows**      | C:\Users\[UserName]\AppData\Local\csvcubed\csvcubed-cli\Logs\out.log |
+| **Linux**        | /home/[UserName]/.cache/csvcubed/csvcubed-cli/Logs/out.log           |
+| **MAC**          | /home/[UserName]/Library/Logs/csvcubed/csvcubed-cli/Logs/out.log     |
+
+**Log Retention**  
+A new log file is created daily, with backups being saved for 7 days in the same location. Older logs are overwritten in a cyclic manner with the oldest being replaced first.
+
+## Input types
+
+The inspect command takes a CSV data inputs that are provided in one of two specialised form of tidy data: [canonical approach](./../guides/shape-data.md#canonical-shape) and [pivoted approach](./../guides/shape-data.md#pivoted-shape). More specifically, the command takes the following Qube types:
```

Comment:
I've renamed the `canonical shape` to the `standard shape` everywhere else so needs to be updated here.